### DISCUSSION TITLE
My application gets instantly rejected.

### DISCUSSION THREAD

**Discussion by yikafu** [community/community#49604] [GitHub Education] [answered]
I am a college student in China. I have submitted various documents on GitHub Education to prove that I am a student, including student ID card, third-party certification, school admission letter, etc., but I have been rejected by the robot. I have tried various methods.The following is the reason for rejection I have been getting. I hope it can be solved and I can get the GitHub Student Developer Pack.
![15be6f950b5d72cfe0b8a9d0d5ae2c4](https://user-images.githubusercontent.com/98627904/223974536-5d362b7f-ee79-4c8b-94db-17eae1b30346.jpg)

**Comment by samus-aran** [comment]
Hi @yikafu 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. As you have an academic email we would recommend you select this option and also submit evidence that is positioned higher rather than lower in that option list.

Let us know how you get on. 👍

**Comment by yikafu** [comment, marked as answer]
I have solved this problem.
Before my application was successful, my primary email address was Google Email and my secondary email address was student email.
I thought I had made the changes, but maybe I hadn't saved the changes before.😂
So I set my student email as the main one, I added bio for myself, added my name, and applied again.
My application was approved.
